common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1408,3 +1417,30 @@ def clean_id(self, padding_char='='):
         return "course_{}".format(
             b32encode(unicode(self.location.course_key)).replace('=', padding_char)
         )
+
+    @property
+    def teams_enabled(self):
+        """
+        Returns whether or not teams has been enabled for this course.
+        """
+        if self.teams_configuration:
+            return 'topics' in self.teams_configuration and len(self.teams_configuration['topics']) > 0
+        return False
+
+    @property
+    def teams_max_size(self):
+        """
+        Returns the max size for teams if teams has been configured, else None.
+        """
+        if self.teams_enabled:
+            return self.teams_configuration.get('max_team_size', None)
+        return None
+
+    @property
+    def teams_topics(self):
+        """
+        Returns the topics that have been configured for teams for this course, else None.
+        """
+        if self.teams_enabled:
```

Comment:
In fact, I think you should get rid of it. I think this method should always return the teams topics, whether or not teams are "enabled". Suppose in the future we have a boolean in the `teams_configuration` which signifies whether or not teams are enabled. This method should still be able to get us the teams topics for the course, even if teams are "disabled".